tests/csm/rest/test_capacity_quota.py

```diff
@@ -2264,20 +2090,9 @@ def test_43840(self):
         self.log.info("Data size is %s ", data_size)
 
         self.log.info("Step 5: Perform & Verify GET API to get capacity usage stats")
-        resp = self.csm_obj.get_user_capacity_usage("user", self.user_id)
-        assert resp.status_code == HTTPStatus.OK, \
-                "Status code check failed for get capacity"
-        uid = resp.json()["capacity"]["s3"]["users"][0]["id"]
-        t_obj = resp.json()["capacity"]["s3"]["users"][0]["objects"]
-        t_size = resp.json()["capacity"]["s3"]["users"][0]["used"]
-        m_size = resp.json()["capacity"]["s3"]["users"][0]["used_rounded"]
-
-        assert_utils.assert_equals(self.uid, uid, "id is not equal")
-        assert_utils.assert_equals(total_objects, t_obj, "Number of objects not equal")
-        assert_utils.assert_equal(total_size, t_size, "Total Size mismatch found")
-        assert_utils.assert_greater_equal(m_size, total_size, "Total Used Size mismatch found ")
-        assert_utils.assert_greater_equal(m_size, t_size, "used_rounded and used mismatch found ")
```

Comment:
same as above, we need to have this asset, that is not verified in verify_user_capacity